global_constants.py

```diff
@@ -0,0 +1,3 @@
+import os
+ABS_PROJECT_ROOT_PATH = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I would recommend to take a look at `pathlib` package (part of python standard library - it enables much easier and easy to read path manipulation):

```
from pathlib import Path
ABS_PROJECT_ROOT_PATH = Path(__file__).parent.absolut()

# you can also do things like this:
Path(__file__).parent / "subdir" / "subsubdir"
```

Reply:
fixed